app/models/customer.js

```diff
@@ -12,4 +12,22 @@ export default DS.Model.extend({
   credit: DS.attr('string'),
   want: DS.attr('string'),
   notes: DS.attr('string'),
+
+  statusText: function() {
```

Comment:
nitpick: `Ember.computed('status', function() {` is currently preferred to prototype extension syntax